Files/warranty.cfg.example

```diff
@@ -6,13 +6,20 @@ url =
 
 [discover]
 # enable/disable discoveries
-dell = True
-hp = True
-ibm = True
-lenovo = True
-meraki = True
+cisco = True
+dell = False
```

Comment:
Shouldn't all the vendors be set to True?

Reply:
set back to true